bevy_kayak_ui/src/render/unified/font/extract.rs

```diff
@@ -45,20 +45,22 @@ pub fn extract_texts(
     let chars_layouts = font.get_layout(
         CoordinateSystem::PositiveYDown,
         Alignment::Start,
-        Vec2::new(layout.posx, layout.posy + line_height),
-        Vec2::new(layout.width, layout.height),
+        (layout.posx, layout.posy + line_height),
+        (layout.width, layout.height),
         content,
         line_height,
         font_size,
     );
 
     for char_layout in chars_layouts {
+        let position = Vec2::new(char_layout.position.0, char_layout.position.1);
+        let size = Vec2::new(char_layout.size.0, char_layout.size.1);
```

Comment:
```suggestion
        let position = Vec2::from(char_layout.position);
        let size = Vec2::from(char_layout.size);
```

[Reference](https://docs.rs/bevy/latest/bevy/prelude/struct.Vec2.html#impl-From%3C(f32%2C%20f32)%3E)